core/src/main/scala/chisel3/Data.scala

```diff
@@ -645,6 +647,22 @@ trait WireFactory {
   }
 }
 
+object XMR {
```

Comment:
What does XMR stand for?
Is it `CrossModuleReference`? Why not just spell that out then?

Reply:
XMR is a crypto-coin, lol. So we really decide to use this name?